src/request/sign-swap/SignSwapApi.js

```diff
@@ -42,8 +46,9 @@ class SignSwapApi extends BitcoinRequestParserMixin(TopLevelApi) {
                     recipientType: Nimiq.Account.Type.HTLC,
                     flags: Nimiq.Transaction.Flag.CONTRACT_CREATION,
                 }),
-                /** @type {string} */
-                senderLabel: (this.parseLabel(request.fund.senderLabel, false, 'fund.senderLabel')),
+                senderLabel: /** @type {string} */ (this.parseLabel(
+                    request.fund.senderLabel, false, 'fund.senderLabel',
+                )),
```

Comment:
Probably because VSCode with its newer TS version complains that it's invalid.

The old style is not compatible with new TS versions anymore and will have to be changed everywhere when we upgrade.